pkgs/os-specific/linux/ryzen-smu/default.nix

```diff
@@ -0,0 +1,91 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchurl
+, kernel
+}:
+
+let
+  smu_version = "0.1.5";
+
+  # This fork supports ryzen_monitor_ng, which provides a better interface
+  # than the provided monitor_cpu.
+  # Changes have not been merged upstream.
+  smu_src = fetchFromGitLab {
+    owner = "mann1x";
+    repo = "ryzen_smu";
+    rev = "adaf53c77dc68b99643ab7af729eb9c592cf5e01";
+    hash = "sha256-OQyPqNL8Cd7XQfY8YiOVYqOG7RIu5l+wJoK2FDBg5eg=";
+  };
+
+  monitor-cpu = stdenv.mkDerivation (finalAttrs: {
```

Comment:
don' forget to patch this.

https://gitlab.com/leogx9r/ryzen_smu/-/merge_requests/12#note_1700621578